ambry-tools/src/main/java/com.github.ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -179,12 +210,43 @@ static void updateResourceAndPartition(String srcZkString, String srcClusterName
           System.out.println("DryRun: Add Resource " + resource + " with partition " + srcPartitions);
         } else {
           destAdmin.addResource(destClusterName, resource, idealState);
-          destAdmin.rebalance(destClusterName, resource, 3, "", "");
+          destAdmin.rebalance(destClusterName, resource, replicaNumber, "", "");
           System.out.println("Added Resource " + resource + " with partition " + srcPartitions);
         }
       }
     }
     System.out.println("Cluster " + destClusterName + " is updated successfully!");
   }
+
+  /**
+   * Enable or disable a resource in dest cluster.
+   * @param destZkString the cluster's zk string
+   * @param destClusterName the cluster's name
+   * @param resourceName the resource to enable/disable
+   * @param enable enable the resource if true
+   */
+  static void controlResource(String destZkString, String destClusterName, String resourceName, boolean enable) {
```

Comment:
Possible to add tests for the new methods?

Reply:
Not easy, since helix library doesn't expose getMethod for these status.

Reply:
Fair enough.  But it's still good to test the different options to get code coverage.